### ISSUE TITLE
As an app developer, I want checkboxes and radios to be laid out in multiple columns

### ISSUE THREAD

**Issue by haakemon** [Altinn/app-frontend-react#126]

**Comment by olemartinorg** [comment]
@FinnurO Nope, there's no mention of tables here, so this has nothing to do with `Grid`.

See this screenshot for a visual representation of this requirement: https://github.com/Altinn/altinn-studio/issues/5730#issuecomment-927889177
